Makefile

```diff
@@ -156,14 +156,15 @@ bam_lpileup_h = bam_lpileup.h $(htslib_sam_h)
 bam_plbuf_h = bam_plbuf.h $(htslib_sam_h)
 bam_tview_h = bam_tview.h $(htslib_hts_h) $(htslib_sam_h) $(htslib_faidx_h) $(bam2bcf_h) $(htslib_khash_h) $(bam_lpileup_h)
 bedidx_h = bedidx.h $(htslib_hts_h)
+consensus_pileup_h = consensus_pileup.h $(htslib_sam_h)
```

Comment:
Thanks for these updates.

However given the samtools Makefile is entirely an internal thing, I'm wondering why we use this more complex approach compared to something more trivial such as:

```
consensus_pileup.h:  $(htslib_sam.h)
```

Why are we avoiding Make's builtin dependency analysis?  It's something I've never understood the background to.

Reply:
I think if you try that, you'll find that it doesn't have the desired effect of causing _bam_consensus.o_ to be rebuilt if _htslib/sam.h_ gets updated.

Something like the following would do the trick, but IMHO changing the date stamps on the samtools headers would be artificial and confusing:

```make
consensus_pileup.h: $(htslib_sam_h)
	@touch $@
```